ivy/functional/frontends/torch/nn/functional/dropout_functions.py

```diff
@@ -13,3 +13,19 @@ def dropout(input, p=0.5, training=True, inplace=False):
         ivy.inplace_update(input, ret)
         return input
     return ret
+
+
+@to_ivy_arrays_and_back
+def dropout2d(tensor, prob=0.5, training=True, inplace=False):
+    if tensor.ndim < 2:
+        raise ValueError("Feature dropout requires at least 2 dimensions in the input")
+
+    if not training:
+        return tensor
+
+    ret = tensor.dropout2d(prob)
```

Comment:
I used `ivy.dropout2d` because we agreed to do so, should I use `ivy.dropout` instead?

Reply:
> I used `ivy.dropout2d` because we agreed to do so, should I use `ivy.dropout` instead?

Oh sorry I meant `ivy.dropout2d` haha. You have used `tensor.dropout2d` which doesn't exist. You have to use `ivy.dropout2d`